sdk/src/main/java/com/google/cloud/dataflow/sdk/io/TextIO.java

```diff
@@ -295,14 +315,15 @@ private Bound(String name, String filepattern, Coder<T> coder, boolean validate,
         }
 
         if (validate) {
+          checkState(filepattern.isAccessible(), "Cannot validate with a RVP.");
```

Comment:
I do not think this is a useful error message -- how does a user know what a RVP is?